packages/hardhat-router/src/tasks/deploy.ts

```diff
@@ -82,7 +82,7 @@ task(TASK_DEPLOY, 'Deploys the given modules behind a Proxy + Router architectur
     const modulesData = await deployContracts(contracts, hre);
 
     await hre.run(SUBTASK_GENERATE_ROUTER, {
-      router,
+      router: /*_contractName(*/ router /*)*/,
```

Comment:
A comment here on what's going on would be helpful.

Reply:
sorry removed the change entirely